devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -166,7 +170,7 @@ private void createChildButtonDevices(numberOfButtons) {
 
 	for (i in 1..numberOfButtons) {
 		log.debug "Creating child $i"
-		def supportedButtons = ((isIkeaRemoteControl() && i == REMOTE_BUTTONS.MIDDLE) || isIkeaOpenCloseRemote() || isSomfy()) ? ["pushed"] : ["pushed", "held"]
+		def supportedButtons = (isBSM300() || isMSM300()) ? ["pushed","held","double"] : (((isIkeaRemoteControl() && i == REMOTE_BUTTONS.MIDDLE) || isIkeaOpenCloseRemote() || isSomfy()) ? ["pushed"] : ["pushed", "held"])
```

Comment:
Can you break this out to not use the ternary operator so it's a bit easier to read?